lib/ES/Repo.pm

```diff
@@ -251,7 +255,7 @@ sub all_repo_branches {
         my $shas = $repo->tracker->shas_for_repo( $repo->name );
 
         for my $branch ( sort keys %$shas ) {
-            my $log = run( qw(git log --oneline), $shas->{$branch} );
+            my $log = run( qw(git log --oneline -1), $shas->{$branch} );
```

Comment:
I noticed that this step was super slow and this seems to have sped it up a ton.